lib/logbook_client/api.rb

```diff
@@ -35,10 +35,12 @@ def health
       request_with_rescue { Requests::HealthRequest.new }
     end
 
-    def get_documents(collection_id, search_term = '')
+    def get_documents(collection_id, search_term = '', page: nil, per_page: nil)
```

Comment:
I'd use `options = {}` instead... Named parameters tends to increase the parameters' size and force us to change it later to a hash...

Reply:
agreed, adjusted